pkg/controller/controllermanager.go

```diff
@@ -0,0 +1,36 @@
+package controller
+
+import (
+	serviceaccountcontrollers "github.com/openshift/origin/pkg/serviceaccounts/controllers"
+	"k8s.io/kubernetes/pkg/controller"
+)
+
+const (
+	DockercfgTokenDeletedControllerName = "dockercfg-token-deleted-controller"
+)
+
+type ControllerContext struct {
```

Comment:
Add some comments explaining why we have a different context.  I think I see why you've done it now.  Different informer factory, no options, and no use for `avaialbleResources`?